### PR TITLE
Deployment script and Opcache

### PR DESCRIPTION
**Pull request by chris-moreton**
Various changes to the push.sh script. Also enabled opcache.

### CODE REVIEW

File: .bp-config/php/php.ini [hunk 1]
```diff
@@ -0,0 +1,1971 @@
...
+; "auto" detect order is changed accoding to mbstring.language
+; http://php.net/mbstring.detect-order
+;mbstring.detect_order = auto
+
+; substitute_character used when character cannot be converted
+; one from another
+; http://php.net/mbstring.substitute-character
+;mbstring.substitute_character = none
+
+; overload(replace) single byte functio
```
Comment (kalpaitch) [inline .bp-config/php/php.ini]: Would be good to explicitly enable this line please.

Comment (kalpaitch) [inline .bp-config/php/php.ini]: And this line please `opcache.load_comments=1`

Comment (kalpaitch) [inline .bp-config/php/php.ini]: Think it would probably be good to double the default `opcache.max_accelerated_files=4000`

Comment (kalpaitch) [inline .bp-config/php/php.ini]: What do you reckon to doubling the defaults `opcache.memory_consumption=128`??

Comment (kalpaitch) [inline .bp-config/php/php.ini]: Think also for this... `opcache.interned_strings_buffer=8`

Comment (chris-moreton) [inline .bp-config/php/php.ini]: Yep - will do

Comment (chris-moreton) [inline .bp-config/php/php.ini]: No problem

**Pull request merged**